openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -24,53 +22,45 @@ import '../../../../vendor/js/flot/jquery.flot.stack.js';
 import '../../../../vendor/js/flot/jquery.flot.pie.js';
 import { validateEmail, validatePassword } from './account.js';
 import autocompleteInit from './autocomplete';
+// Used only by the openlibrary/templates/books/edit/addfield.html template
 import addNewFieldInit from './add_new_field';
 import automaticInit from './automatic';
-import { getAvailabilityV2,
```

Comment:
I feel like this is being used on Author pages

Reply:
It is, but it's not used directly using the window object. I'm not removing the code here - just how it is globally exposed

If you grep the codebase you'll see getAvailabilityV2 is only referenced inside openlibrary/plugins/openlibrary/js/availability.js so doesn't need to be exported.